distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -46,11 +46,17 @@ javax.net.ssl.trustStoreType=jks
 #
 # Global URL Properties
 #
+
+# The httpsPort, httpPort, and port properties are used to advertise the system externally
+# The internalHttpsPort and internalHttpPort properties are the ports that the system will run on
+
 org.codice.ddf.system.protocol=https://
 org.codice.ddf.system.hostname=localhost
 org.codice.ddf.system.httpsPort=8993
 org.codice.ddf.system.httpPort=8181
 org.codice.ddf.system.port=8993
+org.codice.ddf.system.internalHttpsPort=8993
+org.codice.ddf.system.internalHttpPort=8181
```

Comment:
❓ Wonder if we should default those to `${org.codice.ddf.system.httpsPort}` and `${org.codice.ddf.system.httpPort}`. This would make it less error prone when setting up a single node not fronted by a proxy or load balancer.